main.go

```diff
@@ -243,9 +244,9 @@ func main() {
 		packageName := "cordova"
 		packageName += "@" + configs.CordovaVersion
 
-		log.Printf("Will check for JS package manager, in directory: %s", workDir)
-		packageManager := detectJsPackageManager(workDir)
-		if err := addJsPackages(packageManager, true, packageName); err != nil {
+		packageManager := jspackage.DetectManager(workDir)
+		log.TPrintf("Js package manager used: %s", packageManager)
```

Comment:
does it worth here to use `TPrintf ` it adds the current time prefix for the message. We usually use this logging before and after command runs to highlight the command's runtime.